rpcs3/rpcs3qt/memory_viewer_panel.h

```diff
@@ -34,6 +34,10 @@ class memory_viewer_panel : public QDialog
 	};
 	Q_ENUM(color_format)
 
+	static constexpr u32 id_base = 1;
+	static constexpr u32 id_step = 1;
```

Comment:
What's that supposed to be. Is it used anywhere?

Reply:
by IDM.